var/spack/repos/builtin/packages/mvapich2-gdr/package.py

```diff
@@ -17,10 +16,11 @@ class Mvapich2Gdr(AutotoolsPackage):
     """
 
     homepage = 'http://mvapich.cse.ohio-state.edu'
-    url      = 'http://mvapich.cse.ohio-state.edu/download/mvapich/spack-mirror/mvapich2-gdr/mvapich2-gdr-2.3.4.tar.gz'
+    url      = 'file:///home/packager/spack-related/tar-bundles/mvapich2-gdr/mvapich2-gdr-2.3.5.tar.gz'
```

Comment:
Can you revert this change? Using `file` means that people have to resort to manual downloading the tarball. Is that what you meant?